docs/tutorial/import_export.rst

```diff
@@ -0,0 +1,1021 @@
+in the ``compounds.yaml`` file the following command can be run.
+
+.. code-block:: shell
+
+    (psamm-env) $ git diff model.yaml
+
+The output should look like the following:
+
+.. code-block:: diff
+
+    @@ -5,6 +5,7 @@ compounds:
+       - include: compounds.yaml
+       reactions:
+       - include: reactions.yaml
+    +  - include: mannitol_pathway.yaml
+       media:
+       - include: medium.yaml
+       limits:
+
+This can be done with any file that had changes to make sure that no
+accidental changes are added in along with whatever the desired changes are.
+In this example there should be one line added in the ``model.yaml`` file,
+three compounds added into the ``compounds.yaml`` file, and one exchange
+reaction added into the ``media.yaml`` file along with one change that removed
+glucose from the list of carbon sources in the medium (by changing the lower
+bound of its exchange reaction to zero).
+
+Once the changes are confirmed these files can be added with the Git add
+command.
+
+.. code-block:: shell
+
+    (psamm-env) $ git add compounds.yaml
+    (psamm-env) $ git add medium.yaml
+    (psamm-env) $ git add model.yaml
+
+These changes can then be committed to the repository using the following
+command:
+
+.. code-block:: shell
+
+    (psamm-env) $ git commit -m 'Addition of mannitol utilization pathway and associated compounds'
+
+Now the model has been updated and the changes have been committed. The Git log
+command can be used to view what commits have been made in the repository.
+This allows you to track the overall progress as well as examine what
+specific changes have been made. More detailed information between the commits
+can be viewed using the ``git diff`` command along with the commit ID that you
+want to compare the current version to. This will tell you specifically what
+changes occurred between that commit and the current version.
+
+You can also view a log of the commits in the model by using the following command:
+
+.. code-block:: shell
+
+    (psamm-env) $ git log
+
+This can be helpful for getting an overall view of what changes have been made
+to a repository.
+
+The Git version tracking can also be used with GitHub_, BitBucket_, GitLab_ or
+any other Git hosting provider to share repositories with other people. This
+can enable you to collaborate on different aspects of the modeling
+process while still tracking the changes made by different groups and
+maintaining a functional model.
+
+.. _GitHub: https://github.com/
+.. _BitBucket: https://bitbucket.org/
+.. _GitLab: https://gitlab.com/
+
+
+Using PSAMM to export the model to other Software
+_________________________________________________
+
+If you want to export the model in a format to use with other
+software, that is also possible using PSAMM. The YAML formatted model can be
+easily exported as an SBML file using the following command:
+
+.. code-block:: shell
+
+    (psamm-env) $ psamm-model sbmlexport > Modified_core_ecoli.xml
```

Comment:
This command can now take a file name as a parameter so there is no need to pipe the output to a file. Instead it might be better to just show this as `psamm-model sbmlexport Modified_core_ecoli.xml`.